app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextActions.java

```diff
@@ -114,10 +136,23 @@ private boolean onClickImpl(final View view) {
                     setHeadingAction(3);
                     return true;
                 }
-                /*case R.string.tmaid_common_unordered_list_char: {
-                    runRegularPrefixAction(_appSettings.getUnorderedListCharacter() + " ");
+                case R.string.tmaid_common_unordered_list_char: {
+                    final String listChar = _appSettings.getUnorderedListCharacter();
+                    final String listPrefix = "$1" + listChar + " ";
+                    runPrefixReplaceAction(PREFIX_UNORDERED_LIST, listPrefix, "$1");
+                    return true;
+                }
+                case R.string.tmaid_common_checkbox_list: {
+                    final String listChar = _appSettings.getUnorderedListCharacter();
+                    final String uncheck = "$1" + listChar + " [ ] ";
+                    final String check = "$1" + listChar + " [x] ";
+                    runPrefixReplaceAction(PREFIX_UNCHECKED_LIST, uncheck, check);
                     return true;
-                }*/
+                }
+                case R.string.tmaid_common_ordered_list_number: {
+                    runPrefixReplaceAction(PREFIX_ORDERED_LIST, "$11. ", "$1");
+                    return true;
+                }
```

Comment:
Copied here from Textactions. As these changes are markdown-specific, I have added them here. Plain text will still default to the previous `runRegularPrefixAction` as found in `TextActions`.